docker/docker-compose.2204.510.yaml

```diff
@@ -3,28 +3,27 @@ version: "3"
 services:
 
```

Comment:
@FranzBusch Do you want me to update the API breakage pipeline to use 5.9 as part of this changeset?

Reply:
Yes please

Reply:
👍 updated